### PR TITLE
add ut for cmdinit

### PR DESCRIPTION
**Pull request by AllenZMC**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
add ut for cmdinit
**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by AllenZMC** [comment]
@XiShanYongYe-Chang @RainbowMango

File: pkg/karmadactl/cmdinit/utils/util_test.go [hunk 1]
```diff
@@ -24,3 +25,24 @@ func TestDownloadFile(t *testing.T) {
 		panic(err)
 	}
 }
+
+func TestListFiles(t *testing.T) {
+	tests := []struct {
+		name string
+		path string
+		want []string
+	}{
```
Comment (XiShanYongYe-Chang) [inline pkg/karmadactl/cmdinit/utils/util_test.go:34]: What happens to input a path that does not exist?

File: pkg/karmadactl/cmdinit/utils/util_test.go [hunk 2]
```diff
@@ -24,3 +25,24 @@ func TestDownloadFile(t *testing.T) {
...
+		{
+			name: "files",
+			path: "../options",
+			want: []string{"../options/global.go"},
```
Comment (XiShanYongYe-Chang) [inline pkg/karmadactl/cmdinit/utils/util_test.go:38]: The result of this test case will depend on the file in the `../options` path.

Comment (AllenZMC) [reply pkg/karmadactl/cmdinit/utils/util_test.go:38]: what good advice do you have @XiShanYongYe-Chang

File: pkg/karmadactl/cmdinit/utils/util_test.go (hunk 1 above)
Comment (AllenZMC) [reply pkg/karmadactl/cmdinit/utils/util_test.go:34]: Empty path will panic

File: pkg/karmadactl/cmdinit/utils/util_test.go (hunk 2 above)
Comment (XiShanYongYe-Chang) [reply pkg/karmadactl/cmdinit/utils/util_test.go:38]: I don't have any ideas yet.

File: pkg/karmadactl/cmdinit/utils/util_test.go (hunk 1 above)
Comment (XiShanYongYe-Chang) [reply pkg/karmadactl/cmdinit/utils/util_test.go:34]: Does that have to be taken into account in the method?

**Pull request closed without merging**